spec/models/user_spec.rb

```diff
@@ -80,12 +80,26 @@
     end
 
     context 'user does not have a digital.education or education.gov.uk email' do
-      subject { create(:user, email: 'test@hrmc.gov.uk') }
+      subject { create(:user, email: email) }
 
-      its(:admin?) { should be_falsey }
+      context 'when other doamin' do
+        let(:email) { 'test@hrmc.gov.uk' }
 
-      it "does shows up in User.non_admins" do
-        expect(User.non_admins).to eq([subject])
+        its(:admin?) { should be_falsey }
+
+        it "does shows up in User.non_admins" do
+          expect(User.non_admins).to eq([subject])
+        end
+      end
+
+      context 'when other email looks like "digital.education.gov.uk" doamin' do
+        let(:email) { 'test@digitalaeducation.gov.uk' }
+
+        its(:admin?) { should be_falsey }
+
+        it "does shows up in User.non_admins" do
```

Comment:
Try to describe tests from the perspective of the user
```suggestion
        it 'is not an admin' do
```

Reply:
Should this be changed in all other uses see: https://github.com/DFE-Digital/manage-courses-backend/blob/79da4a30a7dae58011e797c13ae726f66af99ff2/spec/models/user_spec.rb#L67 or https://github.com/DFE-Digital/manage-courses-backend/blob/79da4a30a7dae58011e797c13ae726f66af99ff2/spec/models/user_spec.rb#L77